physics/module_sf_mynn.F90

```diff
@@ -1616,6 +1636,7 @@ SUBROUTINE SFCLAY1D_mynn(                                      &
 
        ! Compute u* without vconv for use in HFX calc when isftcflx > 0           
        WSPDI(I)=MAX(SQRT(U1D(I)*U1D(I)+V1D(I)*V1D(I)), wmin)
+       !tgs - should USTM be separater for dry, icy, wet?
```

Comment:
USTM was a variable put in the scheme by the hurricane community a long time ago. I'm not sure if anyone uses it.